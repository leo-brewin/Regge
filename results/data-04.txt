# defect
 5.329384317975498250E-03

# i  j   d(defect)/d(lsq(i,j))
  1  2  1.763498907588229740E+01
  1  3  1.764904056659852790E+01
  2  3  1.764904056659852790E+01
  4  1 -2.128582351727335780E+01
  4  2 -2.128582351727335780E+01
  4  3 -2.129919132226968560E+01
  5  1 -2.638319609016202320E+00
  5  2 -2.638319609016202330E+00
  5  3 -2.639398538592749940E+00
  4  5  1.685860601980725880E+01
  6  1 -2.639484651599127150E+00
  6  2 -2.639484651599127160E+00
  6  3 -2.649287906547015680E+00
  5  6  1.690002057338266370E+01
  7  1 -2.640009802117732990E+00
  7  2 -2.640009802117732980E+00
  7  3 -2.642527872084996300E+00
  6  7  1.690457959806811020E+01
  8  1 -2.611074929847048110E+00
  8  2 -2.611074929847048110E+00
  8  3 -2.600765721710281510E+00
  7  8  1.685500488715065660E+01
  9  1 -2.110430620850965440E+01
  9  2 -2.110430620850965440E+01
  9  3 -2.113000373491239020E+01
  8  9  1.681608996827577040E+01
  9  4  3.076070613446500320E+01
