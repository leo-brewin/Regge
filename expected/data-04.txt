# defect
 5.329384317975513650E-03

# i  j   d(defect)/d(lsq(i,j))
  1  2  1.763498907588229700E+01
  1  3  1.764904056659852750E+01
  2  3  1.764904056659852750E+01
  4  1 -2.128582351727335730E+01
  4  2 -2.128582351727335730E+01
  4  3 -2.129919132226968520E+01
  5  1 -2.638319609016202330E+00
  5  2 -2.638319609016202320E+00
  5  3 -2.639398538592749970E+00
  4  5  1.685860601980725880E+01
  6  1 -2.639484651599127170E+00
  6  2 -2.639484651599127170E+00
  6  3 -2.649287906547015690E+00
  5  6  1.690002057338266370E+01
  7  1 -2.640009802117732970E+00
  7  2 -2.640009802117732980E+00
  7  3 -2.642527872084996290E+00
  6  7  1.690457959806811020E+01
  8  1 -2.611074929847048110E+00
  8  2 -2.611074929847048110E+00
  8  3 -2.600765721710281510E+00
  7  8  1.685500488715065660E+01
  9  1 -2.110430620850965360E+01
  9  2 -2.110430620850965360E+01
  9  3 -2.113000373491238960E+01
  8  9  1.681608996827577040E+01
  9  4  3.076070613446500220E+01
